modules/nodes-descheduler-configuring-policies.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-configuring-policies_{context}"]
+= Configuring descheduler policies
+
+You can configure which strategies that the descheduler uses to evict Pods.
+
+// TODO: need to be cluster admin?
```

Comment:
good question, I imagine you need cluster admin to install it since it requires cluster role bindings (@ingvagabund can you confirm the operatorhub install still handles this?)

Reply:
I logged into the console as admin and with a bit hacked index image I was able to deploy the operator and the descheduler with all the cluster roles and bindings. Though, for sure I will know once the operator gets published on the operatorhub.

Reply:
And yes, you need to be cluster admin, resp. have permission to create role binding for https://kubernetes.io/docs/tasks/access-kubernetes-api/#extension-apiserver-authenticates-the-request